listeners.go

```diff
@@ -303,13 +303,19 @@ func IsUnixNetwork(netw string) bool {
 // Network addresses are distinct from URLs and do not
 // use URL syntax.
 func ParseNetworkAddress(addr string) (NetworkAddress, error) {
+	return ParseNetworkAddressWithDefaults(addr, "tcp", 0)
+}
+
+// ParseNetworkAddressWithDefaultNetworkPort is like ParseNetworkAddress
+// but allows the default network and port to be specified.
```

Comment:
```suggestion
// ParseNetworkAddressWithDefaults is like ParseNetworkAddress but allows
// the default network and port to be specified.
```

Reply:
Whoops, yeah thanks, that's the name for the function I had in between 😅